src/style.css

```
@@ -0,0 +1,32 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+
+  margin-right: 4px;
+
+  background-image: url("./images/star.svg");
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 > .stars__star:nth-child(1),
```

You can do it shorter
```suggestion
.stars--1 :nth-child(1),
```